poster/styles/styles.scss

```diff
@@ -1,63 +1,477 @@
     grid-column: 1 / 3;
     grid-row: 1 / 3;
-    background: #9E9E9E;
+    margin: 2.5rem;
+}
+
+.names {
+    font-size: 1.5rem;
+    margin: 2rem;
 }
 
+.secondary-details {
+    grid-column: 4 / 5;
+    grid-row: 2 / 3;
+}
+
+.initials {
+    color: #212121;
+    font-size: 17vw;
+    grid-column: 3 / 4;
+    grid-row: 3 / 5;
+    line-height: 13rem;
+    margin: 3rem;
+    order: 1;
+    text-align: center;
+}
+
+// patterns //
+
 .texture {
+    height: auto;
     margin: 0;
+    object-fit: fill;
+    width: 100%;
 }
 
-.names {
-    grid-column: 1 / 3;
-    grid-row: 4 / 5;
-    background: #7E7E7E;
+.triangle-pattern {
+    background: #212121 url("../images/triangle_pattern.png");
+    background-size: 75%;
+    grid-column: 3 / 4;
+    grid-row: 1 / 2;
 }
 
-.initials-texture {
+.noise_pattern {
+    background: url("../images/noise_pattern.png");
+    margin-top: 1rem;
+}
+
+.two-noise_patterns {
+    background: url("../images/noise_pattern.png") no-repeat left,
+        url("../images/light_noise_pattern.png") no-repeat right;
+    background-size: 50%;
+    grid-column: 4 / 5;
+    grid-row: 3 / 4;
+}
+
+.vertical-dark_noise {
+    background: url("../images/dark_noise_pattern.png");
+    background-size: 75%;
+    grid-column: 2 / 3;
+    grid-row: 3 / 4;
+}
+
+.gradient-triangle_pattern {
+    background: #212121 url("../images/gradient_triangle.png");
+    background-size: 80%;
+    grid-column: 1 / 2;
+    grid-row: 3 / 4;
+}
+
+.noise-horizontal_pattern {
+    background: url("../images/noise_pattern.png") no-repeat left, url("../images/horizontal_stripes.png") no-repeat right;
+    background-size: 50%;
+    grid-column: 3 / 4;
+    grid-row: 2 / 3;
+}
+
+.dot-background {
+    background: #212121 url("../images/dot_pattern.png");
+    background-size: cover;
     grid-column: 3 / 4;
     grid-row: 3 / 5;
-    background: #E2E2E2;
-    color: #212121;
 }
 
-.primary-details {
+.layer-pattern {
+    background: #212121 url("../images/layer-pattern.png");
+    background-size: cover;
+    margin-bottom: 1rem;
+}
+
+.dark-noise_pattern {
+    background: #212121 url("../images/dark_noise_pattern.png") repeat;
+    background-size: 50%;
+}
+
+// grids-flex //
+
+.primary-details_flex {
+    display: flex;
+    flex-direction: column;
     grid-column: 4 / 5;
     grid-row: 1 / 2;
-    background: #E0E0E0;
 }
 
-.secondary-details {
+.tertiary-details_flex {
+    display: flex;
+    flex-direction: column;
     grid-column: 4 / 5;
-    grid-row: 2 / 3;
-    background: #7E7E7E;
+    grid-row: 4 / 5;
 }
 
-.tertiary-details {
-    grid-column: 4 / 5;
+.names_flex {
+    display: flex;
+    flex-direction: column;
+    grid-column: 1 / 3;
     grid-row: 4 / 5;
-    background: #9E9E9E;
+    margin-top: 2rem;
 }
 
-.remove-list {
-    list-style: none;
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
+.flex-grid {
+    flex: 1;
+}
+
+// shapes // 
+
+.shape {
+    position: absolute;
+    z-index: 9000;
```

Comment:
`z-index` is, in part, relative to other `z-index` values in the same CSS. So, it's unnecessary to choose this kind of random number in this instance. A `z-index` of `1` or `10` would also suffice and could be adapted down the road, if necessary.